src/io/CAN.cpp

```diff
@@ -0,0 +1,13 @@
+#include <stdint.h>
+
+#include <EVT/io/CAN.hpp>
+
+namespace EVT::core::IO {
+
+CAN::CAN(Pin txPin, Pin rxPin, bool loopbackEnabled) {
+    this->txPin = txPin;
+    this->rxPin = rxPin;
+    this->loopbackEnabled = loopbackEnabled;
```

Comment:
Depending on your stylistic preferences, I suppose.
```suggestion
CAN::CAN(Pin txPin, Pin rxPin, bool loopbackEnabled) :
    txPin(txPin),
    rxPin(rxPin),
    loopbackEnabled(loopbackEnabled) {
```

Reply:
I think your method is generally the preferred one, I'll switch to that